packages/search-field/src/SearchField.js

```diff
@@ -212,8 +213,18 @@ class SearchField extends React.Component {
     const { focused, selectedItem } = this.state;
 
     const Input = selectedItem ? ReadOnly : EnhancedInput;
+
     Input.displayName = "Input";
 
+    // When there's an item selected and we click the edit icon,
+    // the input ref points to the StaticInput instance, which has no focus().
+    // By setting the component focused prop, the regular
+    // input will get focus when it gets mounted.
+    const autoFocus =
+      this.input !== undefined &&
+      this.input.nodeName !== "INPUT" &&
```

Comment:
I believe this condition should be:
```
this.input && this.input.nodeName !== 'INPUT' && !selectedItem
```

An error could be thrown if `this.input = null` on line 225